hack/build-images.sh

```diff
@@ -138,6 +138,9 @@ image openshift/origin-sti-builder           images/builder/docker/sti-builder
 image openshift/origin-f5-router             images/router/f5
 image openshift/node                         images/node
 
+# images used for tests
+image openshift/test-multicast               images/testing/multicast
```

Comment:
If we are not running this on every PR this should be envar-gated or something, else we will be paying the penalty for building it on every PR anyway.